command/snapshot/inspect/snapshot_inspect.go

```diff
@@ -48,15 +63,35 @@ func (c *cmd) Run(args []string) int {
 		return 1
 	}
 
+	//Set the last* argument to the file name
+	file = args[numArgs-1]
 	// Open the file.
 	f, err := os.Open(file)
 	if err != nil {
 		c.UI.Error(fmt.Sprintf("Error opening snapshot file: %s", err))
 		return 1
 	}
-	defer f.Close()
 
-	meta, err := snapshot.Verify(f)
+	if c.enhance {
+		file, _, err := snapshot.Read(nil, f)
+		if err != nil {
+			c.UI.Error(fmt.Sprintf("Error reading snapshot: %s", err))
+		}
+		stats, offset, err := enhance(file)
+		if err != nil {
+			c.UI.Error(fmt.Sprintf("Error verifying snapshot: %s", err))
```

Comment:
This error message could be updated, it's from the old call to `snapshot.Verify()`